### ISSUE TITLE
탑모델 훈련

### ISSUE THREAD

**Issue by FennexFox** [FennexFox/dlthon#4]
01.18 오후 1시 43분
> 비교적 작은 데이터세임에도 50번의 에포크 이내에 과대적합이 시작되지 않는 것은 모델이 지나치게 작기 때문일 수 있음. 탑모델의 Dense층을 2배로 늘리는 것 검토
> 검증손실의 값이 매우 불안정함: validation set 크기가 지나치게 작은 것이 원인일 수 있음. K-겹 검증 알고리즘 적용

**Comment by FennexFox** [comment]
데이터 증강 패턴을 추가하고 layers.Rescaling을 추가한 커밋의 결과로 val_loss가 아래와 같이변함

<커밋 전>
![Image](https://github.com/user-attachments/assets/484a6576-5bf0-4ead-a5ff-9723fa671d55)

<커밋 후>
![Image](https://github.com/user-attachments/assets/66e630a4-3502-48db-9bf2-3ad2b03c6e90)

이에 대한 추가 실험이 필요함

**Comment by FennexFox** [comment]
utils 모듈에 image_preprocessing 함수 저장
